### PR TITLE
Mpopov/indigo navdrawer

### PR DESCRIPTION
**Pull request by desig9stein**
### Additional information (check all that apply):
 - [ ] Bug fix
 - [ ] New functionality
 - [ ] Documentation
 - [ ] Demos
 - [ ] CI/CD

### Checklist:
 - [ ] All relevant tags have been applied to this PR
 - [ ] This PR includes unit tests covering all the new code ([test guidelines](https://github.com/IgniteUI/igniteui-angular/wiki/Test-implementation-guidelines-for-Ignite-UI-for-Angular))
 - [ ] This PR includes API docs for newly added methods/properties ([api docs guidelines](https://github.com/IgniteUI/igniteui-angular/wiki/Documentation-Guidelines))
 - [ ] This PR includes `feature/README.MD` updates for the feature docs
 - [ ] This PR includes general feature table updates in the root `README.MD`
 - [ ] This PR includes `CHANGELOG.MD` updates for newly added functionality
 - [ ] This PR contains breaking changes
 - [ ] This PR includes `ng update` migrations for the breaking changes ([migrations guidelines](https://github.com/IgniteUI/igniteui-angular/wiki/Update-Migrations))
 - [ ] This PR includes behavioral changes and the feature specification has been updated with them

### CODE REVIEW

**Comment by didimmova** [comment]
Wouldn't it be a better approach for the light theme to use the igx-contrast-color() function or at least the surface color like in the dark theme instead of pure white?

**Comment by StefanIvanov** [comment]
I think @didimmova makes a very good recommendation here, @desig9stein could you implement the styles via the recommended approach?

**Comment by desig9stein** [comment]
> I think @didimmova makes a very good recommendation here, @desig9stein could you implement the styles via the recommended approach?

@StefanIvanov There are two approaches here, one with the contrast function and one with the surface color. I personally think that using the surface color for text color is wrong for both dark and light versions. I am ok with the contrast function approach.

**Comment by StefanIvanov** [comment]
> > I think @didimmova makes a very good recommendation here, @desig9stein could you implement the styles via the recommended approach?
> 
> @StefanIvanov There are two approaches here, one with the contrast function and one with the surface color. I personally think that using the surface color for text color is wrong for both dark and light versions. I am ok with the contrast function approach.

The contrast color was created particularly for text on background scenarios and I also felt it to be more appropriate, let's use that one.

File: projects/igniteui-angular/src/lib/core/styles/components/navdrawer/_navdrawer-theme.scss [hunk 1]
```diff
@@ -183,7 +185,9 @@
         transition: width, transform;
         transition-timing-function: $ease-out-quad, $ease-out-quad;
         box-shadow: --var($theme, 'shadow');
-        border-#{$right}: 1px solid --var($theme, 'border-color');
+        @if $variant != 'indigo-design'{
```
Comment (simeonoff) [inline projects/igniteui-angular/src/lib/core/styles/components/navdrawer/_navdrawer-theme.scss:188]:
```suggestion
        @if $variant != 'indigo-design' {
```

**Pull request merged**